### ISSUE TITLE
Add sound cues when player's height changes

### ISSUE THREAD

**Issue by boholder** [minecraft-access/minecraft-access#214]
One for up, one for down.
How to make continuous up and down sounds natural?

**Comment by c469591** [comment]
Could you please clarify if this section refers to the need for sound effects when players are either in the air or on the ground? I believe a new feature should be added, incorporating sound effects and voice narration while a player is flying, as well as when they land. I've only just started playing and initially, it was quite difficult for me to discern whether I was airborne or on the ground.

**Comment by boholder** [comment]
> Could you please clarify if this section refers to the need for sound effects when players are either in the air or on the ground?

Nope, it's for height changes, if you stop at certain height, there is no continuous sound cue to bother you.
This feature is mainly for letting player know when they step up and down (for example one block higher or lower), or falling down from high place, or being ejected up by some mechanisms.

> a new feature should be added, incorporating sound effects and voice narration while a player is flying

Sure, I'll see if I can continue play the [flying sound when flying with a pair of Elytra](https://minecraft.wiki/w/Elytra#Sounds) when player is flying. 
And I'll see if I can use this sound as this feature's up and down sound too, with different pitches.

> as well as when they land

Once you landed, there is no flying sound, so you know you're on the ground.
